ContactsApp/ContactsApp/ContactsControllers/ContactViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  ContactViewController.swift
+//  ContactsApp
+//
+//  Created by Apple Macbook Pro 13 on 13.08.22.
+//
+
+import UIKit
+// MARK: Protocol
+protocol UpdateContact {
```

Comment:
if this protocol is delegate, use related naming:
`UpdateContantDelegate`
or if it's a protocol use:
`UpdateContantProtocol` or `UpdateContantInterface`